### ISSUE TITLE
Power Platform Import Data - verbose output

### ISSUE THREAD

**Issue by crsnt** [microsoft/powerplatform-build-tools#784]
Hi - is it possible to have verbose output on import data?

A lot of times when things don't import properly, there is no error on the DevOps output why 

e.g. 
Entity Product, Completed Importing 0 of 17, 8 queued for second pass updates'

We have to try using pac tool cmt manually to see the error.

**Comment by petrochuk** [comment]
yes, there is documented flag [--verbose](https://learn.microsoft.com/en-us/power-platform/developer/cli/reference/data#--verbose--v)

**Comment by crsnt** [comment]
Sorry i meant for the devops buils tool action. There is no verbose flag over there
